test/Microsoft.AspNet.Razor.Test/TestFiles/CodeGenerator/Source/AddTagHelperDirective.cshtml

```diff
@@ -1 +1 @@
-﻿@addTagHelper "**??**??*, nice"
\ No newline at end of file
+﻿@addTagHelper something, nice
```

Comment:
With how the tests are setup it doesn't matter about the value of the `@addTagHelper` as long as it can be parsed. If you want more detail we can talk offline :smile: